File: pratice.py/sister.py/mood.c
#include<stdio.h>
#include<string.h>

int main(){
char name[99];
printf("Your Name Dear: ");
scanf("%99[^\n]",name);

char mood[99];
printf("What Is Your Mood Right Now Dear \n %s happy or sad???\n ",name);
scanf("%s",mood);

char happy[] ="Happy";
char sad[] = "sad";

if (strcmp(mood,sad) == 0)
{
    char ans[99];
   printf("What Happend Dear %s, Why your felling Bad??\n \t answer me: ");
   scanf("%s",ans);
   printf("\nDon't Worry Dear %s Just smile And Fogot the Bad Things and Enjoy The Movement \n why you spoiling the Present Movement ?? \n",name);
}
else if (strcmp(mood,happy) == 0)
{
    printf("Thats Goog My Dear %s \n \t Keep Smiling ",name);
}
else
{
    printf("invalid Input Dear %s", name);
}

    return 0;
}

File: pratice.py/sister.py/mood.py
import pyttsx3

def say(self):
    engine = pyttsx3.init()
    voices = engine.getProperty('voices')
    engine.setProperty('voice',voices[0].id)
    engine.setProperty('rate',140)
    engine.setProperty('volume',1.0)
    engine.say(self)
    engine.runAndWait()
    engine.stop()

def mood():
    def save_file(name , ans):
        filename = name + ".txt"
        with open(filename, "w") as f:
            f.write("The Answer of User Is Following>>>>> \n\n")
            f.write(ans)

    say("Enter Your Name Dear")
    name = input("Enter Your Name Dear: ")
    

    say(f"What Is Your Mood Right Now Dear {name} . Happy or Sad??? ")
    print(f"\nWhat Is Your Mood Right Now Dear {name}.Happy or Sad??? ")
    say("Type here")
    mood = input(f"\t Type here: ")

    mood.lower()

    if (mood == "sad"):
        say(f"What Happend Dear {name} Why are you Felling Sad")
        print(f"What Happend Dear {name} Why are you Felling Sad")
        say("Give Me Answer ")
        ans = input("\t Give Me Answer: ")
        save_file(name,ans)
        say(f"Don't Worry, Dear {name}. Everything will be right. Just forget the past and Keep Smiling")
        print(f"\nDon't Worry, Dear {name}. Everything will be right. \n Just forget the past and Keep Smiling")

    elif (mood == "happy"):
        say(f"That's Good My Dear {name} Keep Smiling")
        print(f"\nThat's Good My Dear {name} \n Keep Smiling")

    else:
        say(f"Invalid Input!!!")
        print(f"\nInvalid Input!!!")



mood()